src/components/utils/src/date_time.cc

```diff
@@ -46,6 +46,7 @@ namespace date_time {
 TimeDuration getCurrentTime() {
   return bpt::microsec_clock::local_time() - bpt::from_time_t(0);
 }
+// cppcheck-suppress unusedFunction //Used in unit tests
```

Comment:
can all three (`TimeDurationZero`, `get_just_mSecs`, `get_just_uSecs`) functions be moved into an `ifdef BUILD_TESTS` block?

Reply:
Done in 9f1a645791e598169044d384c2a131addb74c075